spec.bs

```diff
@@ -325,7 +274,29 @@ To <dfn>serialize an integer</dfn>, represent it as a [=string=] of the shortest
 possible decimal number.
 
 Issue: This would ideally be replaced by a more descriptive algorithm in Infra.
-    See infra/201
+    See <a href="https://github.com/whatwg/infra/issues/201">infra/201</a>.
+
+To <dfn algorithm export>append an entry to the contribution cache</dfn> given a
+[=contribution cache entry=] |entry|:
+1. [=list/Append=] |entry| to the [=contribution cache=].
+
+To <dfn algorithm export>mark a batching scope complete</dfn> given a [=batching
```

Comment:
I wonder whether we should make it more clear in the algorithm name that a report may be created and scheduled.

Reply:
Went with "Process contributions for a batching scope"